tests/lib/devicetree/api/src/main.c

```diff
@@ -41,6 +41,13 @@
 #define TO_STRING(x) TO_STRING_(x)
 #define TO_STRING_(x) #x
 
+/*
+ * Since dependency ordinals are unique, we can use them to make
+ * sure two node identifiers refer to the same node.
+ */
+#define SAME_NODE(node_id1, node_id2) \
```

Comment:
Should we have `DT_SAME_NODE()` generally available so we don't need to replicate this (or have people try to do the same thing with `DT_REG_ADDR()` which isn't necessarily unique)?

Reply:
> Should we have `DT_SAME_NODE()` generally available

This is in master now, so I'll just use it.